### PR TITLE
Week One Progress for LendIt

### PR DESCRIPTION
**Pull request by TheFlash98**

### CODE REVIEW

File: _soc_projects/55-nihal-lendit.markdown [hunk 1]
```diff
@@ -40,4 +40,9 @@ The repository can be found [here](https://github.com/codemaxx/LendIt).
 ### Week 0
 * In dept discussions were conducted about the details of the project and every single feature.
 * Relevant APIs were searched for.
-* The structure of the database was finalised.
\ No newline at end of file
+* The structure of the database was finalised.
+
+### Week 1
+* Used the APIs, get familiar with them.
```
Comment (nihal111) [inline _soc_projects/55-nihal-lendit.markdown]: get -> got

Comment (TheFlash98) [inline _soc_projects/55-nihal-lendit.markdown]: Oops

**Comment by TheFlash98** [comment]
I've done @grubdragon work as well, as he is out of town. He messaged the content.

**Comment by nihal111** [comment]
Thanks a lot. Meged.

**Pull request merged**